### PR TITLE
NO TICKET adapt detekt config

### PR DESCRIPTION
**Pull request by mac641**
- Check detekt rules and set to true if useful
- Apply latest detekt rules changes
- Add Doc comments if necessary
- Minor refactoring

### CODE REVIEW

File: app/src/androidTest/java/de/hsaugsburg/teamulster/sohappy/detector/FerTFLiteSmileDetectorImplTest.kt [hunk 1]
```diff
@@ -35,7 +31,7 @@ class FerTFLiteSmileDetectorImplTest {
         scenario.onActivity {
             val istr = instrumentationContext.assets.open("tflite_test.jpg")
             val detectResult = tfliteImpl.detect(BitmapFactory.decodeStream(istr))
-            assert(detectResult.isSmiling)
+            assertEquals(true, detectResult.isSmiling)
```
Comment (AnnaGsell) [inline app/src/androidTest/java/de/hsaugsburg/teamulster/sohappy/detector/FerTFLiteSmileDetectorImplTest.kt:34]: why not assertTrue for checking a boolean condition?

Comment (AnnaGsell) [inline app/src/androidTest/java/de/hsaugsburg/teamulster/sohappy/detector/FerTFLiteSmileDetectorImplTest.kt:34]: why not assertTrue for checking a boolean condition?

File: app/src/androidTest/java/de/hsaugsburg/teamulster/sohappy/detector/FerTFLiteSmileDetectorImplTest.kt [hunk 2]
```diff
@@ -44,7 +40,7 @@ class FerTFLiteSmileDetectorImplTest {
         scenario.onActivity {
             val istr = instrumentationContext.assets.open("tflite_negative_test.png")
             val detectResult = tfliteImpl.detect(BitmapFactory.decodeStream(istr))
-            assert(!detectResult.isSmiling)
+            assertEquals(false, detectResult.isSmiling)
```
Comment (AnnaGsell) [inline app/src/androidTest/java/de/hsaugsburg/teamulster/sohappy/detector/FerTFLiteSmileDetectorImplTest.kt:43]: same here

**Review by AnnaGsell — CHANGES_REQUESTED** [review]
nice job, but still have a look a the comments

**Review by craftamap — APPROVED** [review]

**Review by diehmlukas — APPROVED** [review]

**Review by AnnaGsell — APPROVED** [review]

**Pull request merged**